apps/microtvm/arduino/template_project/microtvm_api_server.py

```diff
@@ -43,6 +43,19 @@
 
 IS_TEMPLATE = not (API_SERVER_DIR / MODEL_LIBRARY_FORMAT_RELPATH).exists()
 
+# Maps a short, identifying microtvm device string to (target, arduino_board).
+MICRO_DEVICES = {
```

Comment:
OK for me to move it into the server side. In fact I'm planing to add that list as the choices for PROJECT_OPTIONS '{zephyr,arduino}_board` option because currently the API doesn't do that. That's necessary for the TVMC + micro targets work. So that change concur with what I'm planning.

Reply:
BTW, the confusion with "platform" in my understanding doesn't happen with the Arduino platform, because `due`, `nano33ble`, etc are indeed termed as boards by the Arduino project, hence one additional reason to s/MICRO_DEVICES/BOARDS/.

Reply:
i might be a bit dense here but where is this used in microtvm_api_server.py? prefer to avoid complicating that file except when necessary

Reply:
I added the model info for each board to BOARD_PROPERTIES for Arduino and added BOARD_PROPERTIES for zephyr.

Reply:
I like this change - looks good for now. In a future PR, I would like to revisit how boards are specified, to make it easier for users to use boards that aren't explicitly in `BOARD_PROPERTIES`.